debugger/debugrt/host/host.c

```diff
@@ -47,14 +47,28 @@ static bool raise_debugger_events()
         {
             // If specified, override oe_debugger_contract_version from the
             // environment.
+#ifdef _WIN32
+            char* version = NULL;
+            size_t size = 0;
+            _dupenv_s(&version, &size, "OE_DEBUGGER_CONTRACT_VERSION");
+#else
             char* version = getenv("OE_DEBUGGER_CONTRACT_VERSION");
+#endif
```

Comment:
Could we use `getenv_s` which is available in C11 on both Windows and Linux?

Alternatively you could use `oe_dupenv()`. In general I would prefer to avoid ifdefs when possible.

Reply:
getenv_s() works fine in this update, and if oe_dupenv() works I'll update it again.

Reply:
getenv_s is fine. Thanks!